ci/scripts/docker.sh

```diff
@@ -6,6 +6,7 @@ set -euo pipefail
 ghcraddr="ghcr.io/risingwavelabs/risingwave"
 dockerhubaddr="risingwavelabs/risingwave"
 arch="$(uname -m)"
+CARGO_PROFILE=${CARGO_PROFILE:-production}
```

Comment:
It will still default to `production`, only in the fast docker build we will use `release` profile.